FFernando | QA Internship | Sprint 6 | User Story Test Automations

xXFranzFXx opened the pull request:
Franz Fernando 
QA Internship Sprint 6

Test automations for Koel user stories:
[INTERNSHIP-83417 | Create New Smart Playlist](https://track.testpro.io/browse/INTERNSHIP-83417l)
[INTERNSHIP-83469 | Add Songs to the Smart Playlist](https://track.testpro.io/browse/INTERNSHIP-83469)
CLONED BY INTERNSHIP-83449 |  Koel App | Artists

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
creates addSongToPlayist test case

xXFranzFXx committed:
syntax corrections for chromedriver

xXFranzFXx committed:
homework 18 initial commit

xXFranzFXx committed:
sets up @Test playSong method, configures TestNG xml

xXFranzFXx committed:
sets up helper methods for playSong test case

xXFranzFXx committed:
creates playSong testCase

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
updates homework 19

xXFranzFXx committed:
creates deletPlaylist test case

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
refactors homework 19 with selenium waits

xXFranzFXx committed:
refactors basetest  class

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
creates renamePlaylist method

xXFranzFXx committed:
Creates BasePage Object

xXFranzFXx committed:
Creates HomePage object

xXFranzFXx committed:
creates login test using POM

xXFranzFXx committed:
Syntax corrections

xXFranzFXx committed:
refactors BasePage to SPF

xXFranzFXx committed:
refactors LoginPage to SPF

xXFranzFXx committed:
Refactors HomePage and LoginPage for SPF

xXFranzFXx committed:
reuses homework22 for SPF test case

xXFranzFXx committed:
initial commit

sets up selenium grid

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
initial commit

sets up lambdatest

xXFranzFXx committed:
homework-25

sets up lambdatest for selenium parallel testing

xXFranzFXx committed:
syntax corrections

removes unnecessary files, makes syntax corrections to BaseTest, makes syntax corrections to BasePage

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
creates Jenkins xml

xXFranzFXx committed:
initial commit

sets up cucumber, gherkin
sets up test scenario

xXFranzFXx committed:
corrects cucumber-gherkin version to 7.9.0

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
updates LoginTests

Adds test cases to LoginTests (Smoke)

xXFranzFXx committed:
updates HomePage class

modifies HomePageTests, HomePage, LoginPage

xXFranzFXx committed:
updates LoginPage

syntax correction on build.gradle

xXFranzFXx committed:
Adds dataprovider

adds excel file for external data provider, changes AlbumsPage, LoginTests, BaseTest, adds Jenkins xml file

xXFranzFXx committed:
Update SmokeTests.xml

xXFranzFXx committed:
updates Homepage

xXFranzFXx committed:
Creates AllSongsPage

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
updates Homepage

xXFranzFXx committed:
updates HomPageTests

syntax correctin on HomePage

xXFranzFXx committed:
updates pages

xXFranzFXx committed:
adds ProfileTests

xXFranzFXx committed:
Adds AllSongsPage

adds AllSongsTests

xXFranzFXx committed:
Adds ArtistPage

creates ArtistsTests
syntax corrections

xXFranzFXx committed:
Adds AlbumsPage

creates AlbumsTests

xXFranzFXx committed:
updates HomeTests

syntax changes  build.gradle file

xXFranzFXx committed:
Syntax Corrections AlbumsPage

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
Adds new locators to Homepage for Info panel

xXFranzFXx committed:
Adds test cases for Jenkins to HomeTests

adds test cases for Info panel user story Sprint-9

xXFranzFXx committed:
Errors

debugging errors "Caused by: java.lang.IllegalArgumentException: Input must be set"

xXFranzFXx committed:
syntax changes

xXFranzFXx committed:
Syntax changes

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
Creates InfoPanel test cases

xXFranzFXx committed:
creates LogoutTests automation test cycle for Koel log out user story

creates test for Sprint-9 koel log out user story acceptance criteria

xXFranzFXx committed:
updates test cases

xXFranzFXx committed:
creates new test cases for login tests

xXFranzFXx committed:
updates login tests

xXFranzFXx committed:
adds log output for user story tests

xXFranzFXx committed:
syntax change to configuration file

xXFranzFXx committed:
changes login tests

all login tests have been verified

xXFranzFXx committed:
adds test descriptions to testng attributes

xXFranzFXx committed:
adds descriptions to all Test attributes

xXFranzFXx committed:
removes comments

xXFranzFXx committed:
changes project structure

Begins updating cucumber tests to run with BaseTest class

xXFranzFXx committed:
Adds new cucumber stepDefinitions for User story

Created Logout tests
Setups parallel testing for cucumber

xXFranzFXx committed:
syntax corrections

change project structure

xXFranzFXx committed:
Syntax corrections

xXFranzFXx committed:
Adds login feature for cucumber tests

User story Log in features

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
updated test cases

added profile page test case

xXFranzFXx committed:
syntax correction

xXFranzFXx committed:
adds profile feature for cucumber

adds profile theme test

xXFranzFXx committed:
updates profile feature

xXFranzFXx committed:
updates basedefinitions

fixes error with profile feature

xXFranzFXx committed:
adds profilepage class

xXFranzFXx committed:
Adds homepage feature file

xXFranzFXx committed:
updates stepdefinitions

xXFranzFXx committed:
creates homepage user story test automations

Cucumber homepage feature and stepdefinitions

xXFranzFXx committed:
new branch

xXFranzFXx committed:
adds logging

modifies project structure

xXFranzFXx committed:
adds screenshots to extent reports

xXFranzFXx committed:
updates logging

xXFranzFXx committed:
Refactor all tests

xXFranzFXx committed:
updates gradle dependency for extent reports

xXFranzFXx committed:
adds env

xXFranzFXx committed:
updates baedefinitions

xXFranzFXx committed:
fixes extent reports

xXFranzFXx committed:
Update LoginTests.java

xXFranzFXx committed:
changes deleteplaylist test

xXFranzFXx committed:
updates hometests

xXFranzFXx committed:
updates extent reports

xXFranzFXx committed:
updates tests

xXFranzFXx committed:
updates cucumber basedefinition

xXFranzFXx committed:
corrects deleteAllPlaylists test

xXFranzFXx committed:
adds test cases for allsongs

xXFranzFXx committed:
updates homepage tests

xXFranzFXx committed:
copies dbfiles

xXFranzFXx committed:
Update build.gradle

xXFranzFXx committed:
Update .env

xXFranzFXx committed:
updates tests for sprint 1

xXFranzFXx committed:
adds user story tests

Account creation user story tests

xXFranzFXx committed:
deletes unnecessary files

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
Syntax corrections

xXFranzFXx committed:
updates extent reports

xXFranzFXx committed:
updates testcases

xXFranzFXx committed:
Adds new testcase to AccountCreationTests

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
Adds new test cases to UpdatePasswordTests

xXFranzFXx committed:
deletes unnecessary files

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
delete unnecessary code and files

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
updates extent reports

xXFranzFXx committed:
update gitignore

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
updates registration tests

xXFranzFXx committed:
Update RegistrationPage.java

xXFranzFXx committed:
updates pom

xXFranzFXx committed:
adds testcases

xXFranzFXx committed:
updates testcases

xXFranzFXx committed:
updates testcases

xXFranzFXx committed:
updates testcases

xXFranzFXx committed:
updates favorites tests

xXFranzFXx committed:
changes updateEmail testcases

xXFranzFXx committed:
adds Search testcases

xXFranzFXx committed:
Update HomePage.java

xXFranzFXx committed:
syntax corrections

xXFranzFXx committed:
Update Sprint-2.xml

xXFranzFXx committed:
adds dataprovider

xXFranzFXx committed:
Update DataProviderUtil.java

xXFranzFXx committed:
Update Sprint-2.xml

xXFranzFXx committed:
syntax correction

xXFranzFXx committed:
delete unused code

xXFranzFXx committed:
Update BaseTest.java

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
adds allSongsTests

xXFranzFXx committed:
adds playlists tests and allsongs tests

xXFranzFXx committed:
updates playlist tests

xXFranzFXx committed:
updates locators

corrects issues with albums page test failing due to poor locators

xXFranzFXx committed:
initial commit

xXFranzFXx committed:
updates page models and testcases

xXFranzFXx committed:
corrects smartplaylists tests

deletes unused files / code

xXFranzFXx committed:
syntax correction

xXFranzFXx committed:
delete unused code

xXFranzFXx committed:
updates delete playlist test

xXFranzFXx committed:
Update LoginPage.java

xXFranzFXx committed:
Update .gitignore

xXFranzFXx committed:
remove bin folder from tracking

xXFranzFXx committed:
remove log files

Berichicko commented:
Hello,

I have checked your Sprint C, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![804deb64dc406a20a75da05d91a1a69e](https://user-images.githubusercontent.com/104761482/235961479-9afb2841-b657-46c9-99f8-2402f933b843.jpg)


![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/4745f3cb-fd4f-40c3-836d-9c70a1585172)

xXFranzFXx committed:
updates test configuration

corrects parallel execution of testscases.  modifies baseTest methods to allow tests to run in parallel
